core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -141,6 +142,7 @@ private void setContextInitializers(ContextInitializersXmlElement contextInitial
         // To keep with prev version. A few will probably be moved to the configuration
         initializerList.add(new SdkVersionContextInitializer());
         initializerList.add(new DeviceInfoContextInitializer());
+        initializerList.add(new BuildContextInitializer());
```

Comment:
This depends on whether we plan to submit this code as part of the SDK or just as an example